Tests/FluentAssertions.Specs/Primitives/DateOnlyAssertionSpecs.cs

```diff
@@ -602,6 +612,65 @@ public void When_asserting_subject_null_dateonly_should_not_have_month_should_th
         }
     }
 
+    public class NotBe
+    {
+        [Fact]
+        public void Different_dateonly_values_are_valid()
```

Comment:
Please let me know if you rather just have my raw review instead of giving small hints and thinking aloud.

In addition to Dennis' blog post, I can recommend https://freecontent.manning.com/making-better-unit-tests/

An unusual part about this way of naming methods is that it wants to _avoid_ dictating a strict naming pattern as one easily ends up bending over backwards to fit the scheme.

While `NotBe` should not be directly included the naming, my gut feeling is that `valid` is a little to vague even though we can infer that context from the name of the nested class `NotBe`.

The two first tests could maybe be named
* `different_values_are_unequal`
* `different_nullable_values_are_unequal`

Or is those just tautologies? 🤔

Reply:
Yeah I have problems with the tautologies too and I feel valid gets us closer.

How about "considered" valid?

Reply:
Nah... I don't think it helps adding `considered`.

Unless @dennisdoomen thinks this is worth digging into, I'll approve and merge it as it is.